src/citrine/informatics/design_candidate.py

```diff
@@ -131,3 +131,63 @@ class DesignCandidate(Serializable["DesignCandidate"]):
 
     def __init__(self):
         pass  # pragma: no cover
+
+
+class GenerationResult(Serializable["GenerationResult"]):
+    """A Citrine Generation Design Result.
+
+    This class represents the molecule greated by a generative design execution.
+    """
+
+    seed = properties.String("seed")
+    """The seed used to generate the molecule."""
+    mutated = properties.String("mutated")
+    """The mutated molecule."""
+    fingerprint_similarity = properties.Float("fingerprint_similarity")
+    """The fingerprint similarity between the seed and the mutated molecule."""
+    fingerprint_type = properties.String("fingerprint_type")
+    """The fingerprint type used to calculate the fingerprint similarity."""
+
+    def __init__(self):
+        pass  # pragma: no cover
+
+
+class GenerationExecutionResult(Serializable["GenerationExecutionResult"]):
```

Comment:
We could possibly combine this class and the above class into one for the sake of the SDK. It might be a bit simpler to have less nested items. The combined class could be a called a `GenerativeDesignResult` which contains the meta-data fields and the result fields.

Reply:
I created these two different classes because this matches the return type from backend. Maybe we can talk about how to combine.

Reply:
It should be straightforward. The combined class will look something like

```
class GenerativeDesignResult():
    uid: UUID
    execution_id: UUID
    seed: str
    mutated: str
    ....
```

Reply:
When the api is working, I'll play around with this!